tools/bq-queryplan-sankey/readme.md

```diff
@@ -0,0 +1,111 @@
+# Sankey visualization for BigQuery query plan
```

Comment:
Capitalize filename: `README.md`

Reply:
Also, add an entry in the [top-level README](https://github.com/GoogleCloudPlatform/professional-services/blob/master/README.md) for this tool. Remember to maintain alphabeticla order.

Reply:
Capitalized `README.md`. 
Will wait for discussions with pull request authors #181 and #193 before putting this on top level `README.md`